### PR TITLE
Refactor interview

### PR DESCRIPTION
**Pull request by olivierschultz**
- Restructure preparation
- Add behavioral questions
- Fix typo

### CODE REVIEW

**Comment by olivierschultz** [comment]
@BerthelotAnthony Had to create another PR since I fucked up with the previous one. Could no longer checkout the `interview` branch, I even tried to fetch and pull.

I took into consideration your previous reviews.

File: interview/interview_resume.md [hunk 1]
```diff
@@ -1,110 +1,109 @@
...
+# Interview - Resume
 
-# General
+## General
 
-- [x] My resume fits on one page.
+- [ ] My resume fits on one page.
 
-- [x] I've exported my resume to a pdf.
+- [ ] I've e ported my resume to a pdf.
```
Comment (BerthelotAnthony) [inline interview/interview_resume.md]: Typo ? exported

File: interview/interview_cover_letter.md [hunk 2]
```diff
@@ -26,15 +25,15 @@ Interview - Cover Letter
...
 __Tip__: Writing a reason like, "I am interested in working for a top company" is too vague. Instead, try to describe how it connects to your passions and career goals when explaining why [x company] or [y role].
 
-# Body Paragraphs
+## Body Paragraphs
 
 - [ ] The body paragraphs in my cover letter contain 2 - 3 examples that detail individual work experiences, as opposed to just including descriptions of my general job duties.
 
 - [ ] I include at least one example that describes how I used code or other technical skills to develop a product or solve a problem.
 
 - [ ] I include at least one example showcases my work ethic, communication skills, teamwork and leadership abilities.
```
Comment (BerthelotAnthony) [inline interview/interview_cover_letter.md]: that showcases ?

**Comment by olivierschultz** [comment]
@BerthelotAnthony Can you merge this one once you're done with the review pls?

**Pull request merged**